File: integration_tests/java/user/PreferencesDoNotTrackV0.java
package com.jsontypedef.jtdcodegendemo;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonSubTypes;
import com.fasterxml.jackson.annotation.JsonTypeInfo;
import com.fasterxml.jackson.annotation.JsonValue;

import java.util.List;
import java.util.Map;


public class PreferencesDoNotTrackV0 extends PreferencesDoNotTrack {

  
  @JsonProperty("do_not_track")
  private Boolean doNotTrack;


  
  public PreferencesDoNotTrackV0() {
  }
  


  public Boolean getDoNotTrack() {
    return doNotTrack;
  }

  public void setDoNotTrack(Boolean doNotTrack) {
    this.doNotTrack = doNotTrack;
  }

}

File: integration_tests/csharp/user/PreferencesDoNotTrackV0.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Newtonsoft.Json.Linq;

namespace Jtd.JtdCodegenDemo
{
    /// <summary>
    /// Our pre-GDPR do-not-track settings
    /// </summary>
    
    public class PreferencesDoNotTrackV0 : PreferencesDoNotTrack
    {

        [JsonProperty("version")]
        public const string Version = "v0";


        /// <value>
        /// An all-or-nothing do-not-track setting
        /// </value>
        [JsonProperty("do_not_track")]
        public bool DoNotTrack { get; set; }





    }
}
